D_damagedW/D_damagedW.ipynb

```diff
@@ -1,5 +1,23 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This validation case is NRA measurements of D in self-damaged W performed by [S. Marlekj et al.](https://www.sciencedirect.com/science/article/pii/S0022311515303470?via%3Dihub)\n",
+    "\n",
+    "The experimental procedure included three main phases. A high-purity polycrystalline W sample of 0.8 mm thickness was pre-damaged with 20 MeV W ions. The pre-damaged sample was then exposed to low-energy (~0.3 eV) D atomic flux of $5.8\\times 10^{18}\\,\\textrm{m}^{-2}\\textrm{s}^{-1}$ at 600 K. The exposure continued until the D fluence of $1\\times 10^{24}\\,\\mathrm{m}^{-2}$ was reached. Finally, an isothermal desorption of D for 43 h. at 600 K was conducted. \n",
+    "\n",
+    "The FESTIM model is based on the approach of [E.A. Hodille et al.](https://iopscience.iop.org/article/10.1088/1741-4326/aa5aa5/meta) Only isothermal D exposure and desorption phases are simulated omitting intermediate cooling/re-heating steps.\n",
+    "\n",
+    "The D diffusivity in W is defnied according to the findings of [N. Fernandez et al.](https://www.sciencedirect.com/science/article/pii/S1359645415003043) Five types of trapping sites are included to reproduce the experimental data: two intrinsic traps and three extrinsic traps with sigmodial distribution ($f$) within the damaged layer:\n",
```

Comment:
Why not use HTM for this? I already notice a - small - discrepency with the original Fernandez 2015 data. The original diffusivity pre-exp factor is 1.9310E-7 m2/s not 1.900E-7 m2/s. 

Plus it should be precised that it's rescaled for deuterium.

Reply:
Etienne sent me $D_0=1.9e-7$ and I automatically wrote that that's Fernandez's data.

Reply:
I also think that, compared to existing cases in FESTIM V&V book, we'd have to add details on surface processes in the description

Reply:
> Etienne sent me D0=1.9E−7 and I automatically wrote that that's Fernandez's data.

Yep I understand. But rigorously that's Fernandez's data modified ;-)

You could have 

```python
import h_transport_materials as htm

fernandez_diff = htm.diffusivities.filter(material=htm.Tungsten, author="fernandez")[0]
F.Material(D_0=fernandez_diff.pre_exp.magnitude, ....)
```
That way no rounding error. 



> I also think that, compared to existing cases in FESTIM V&V book, we'd have to add details on surface processes in the description

Yes probably. But is this for a scientific paper, this repo, or the V&V book?

Reply:
I don't see a point for having different variants for the paper and V&V (or there might be differences?), but I think the description shall contain some words about what processes on a surface are considered. For example, in this example only adsorption and desorption are considered, whereas the case for D implantation in damaged Eurofer also includes an abstraction flux.

Reply:
In general, this repo can include comparison a cross-code comparison, whereas V&V book will have only FESTIM x experiments.

Reply:
> I don't see a point for having different variants for the paper and V&V

Yes correct. I guess a paper usually has even more details.

> but I think the description shall contain some words about what processes on a surface are considered.

I agree